### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Goruchie**
# **Portfolio desktop version**

###  **Add media queries changes:**
- For Headline section.
- For Works section.
- For About Myself section.
- For Footer section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,160 +21,232 @@
         <a class="hamburger" href="#">
```
Comment (mgmediaweb) [inline index.html:21]: Kindly, I suggest you indicate an appropriate **title** for your site

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Goruchie, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost5

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️

*** Kindly, I suggest you revise some details in your design to make it look more like Figma's design.

image

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Goruchie) [reply index.html:21]: @mgmediaweb Thank you! I think I already fixed all the issues, I don't know if the appropriate title which you said is about the meta title of the site, but I fixed that too.

**Review by smunozmo — APPROVED** [review]
Hi @Goruchie, 👋 

## Highlights 👏 

- Excellent job making all the suggested changes, congrats on it! 🎉 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**